generator/uscore/uscore_generator.rb

```diff
@@ -240,7 +242,13 @@ def create_revinclude_test(sequence)
           description: "A Server SHALL be capable of supporting the following _revincludes: #{sequence[:revincludes].join(', ')}"
         }
         search_params = get_search_params(first_search[:names], sequence)
-        revinclude_test[:test_code] = search_params
+        revinclude_test[:test_code] = %(
+               any_provenances = false
+               could_not_resolve_all = []
```

Comment:
Could this be given a more clear name? Maybe something along the lines of `required_search_params`?